end-to-end-test/local/runtime-config/portal.properties

```diff
@@ -263,8 +263,8 @@ ucsc.build=hg19
 oncoprint.defaultview=patient
 
 # OncoPrint driver mutation annotations
- oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
- oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
+ oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotations
+ oncoprint.custom_driver_annotation.tiers.menu_label=Driver tiers
```

Comment:
```suggestion
 oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
```

Reply:
@pvannierop after our discussion with @inodb 
Please follow up now that Bas is gone.